tests/qchem/of_tests/test_convert.py

```diff
@@ -813,3 +813,47 @@ def dummy_ansatz(phis, wires):
     res = dummy_cost(phis)
 
     assert np.abs(res - expected_cost) < tol["atol"]
+
+
+@pytest.mark.parametrize(
+    ("electrons", "orbitals", "singles_ref", "doubles_ref"),
+    [
+        # trivial case
```

Comment:
should we explain origin of this data (h2 in sto6g I presume)?

Reply:
Done.